app/views/posts/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="create-post-c">
+    <h2>Create a post</h2>
+
+    <%= form_with model: @post, url: user_posts_path, scope: :post, html: {class: "post-form-c"} do |form| %>  
+        <%= form.text_field :title, placeholder: "Title", required: true, class: "post-title-c" %>
+        <%= form.text_area :text, placeholder: "Type here...", rows: 20, required: true,  class: "post-text-c" %>
+    <%= form.submit "Post", class: "submit-btn" %>
+    <% end %> 
+</div>
```

Comment:
- [ ] I can't create when ı click on the post button. Kindly fix it.
- [ ] Also to reach the form ı have to type the URL on the address bar. It will be better if you add a form link on the page so users can reach fastly. ### OPTIONAL
![image](https://user-images.githubusercontent.com/38635158/192039755-635f4b8d-8629-4c84-a418-e83f93d4dfd0.png)